exercises/exercise2.js

```diff
@@ -0,0 +1,21 @@
+const employee = ['Juan', 'Angela', 'Carlos', 'Daniela'];
+const role = ['CEO', 'CTO', 'DEV', 'DEV'];
+const seniority = [5, 10, 4, 2];
+
+// Escribir un programa que imprima la información completa por empleado
+for(let i=0; i<employee.length; i++){
+    console.log(employee[i] + ' es el/la ' + role[i] + ' y tiene ' + seniority[i] + ' años de experiencia');
+};
```

Comment:
Remember the built in methods that I explained in the meet:
```suggestion
employee.forEach((item, idx) => {
  console.log(item + ' es el/la ' + role[idx] + ' y tiene ' + seniority[idx] + ' años de experiencia');
})
```

Here, the new thing is the 'idx' param which will save the index of the array.